src/cli/output.py

```
@@ -0,0 +1,229 @@
+        else:
+            console.print(f"  {settings}")
+
+
+def print_db_status(status: dict[str, Any]) -> None:
+    """Print database status."""
+    console.print("\n[bold]Database Status[/bold]")
+    console.print("=" * 40)
+
+    connected = status.get("connected", False)
+    if connected:
+        console.print(f"\nConnection: [green]Connected[/green]")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)